File: 백준/Bronze/2525. 오븐 시계/오븐 시계.py
import sys
input = sys.stdin.readline

def Time(hour: int, minute: int, cookTime: int) -> tuple:
    total_time = ((hour * 60) + minute) + cookTime
    total_time %= 1440
    
    return divmod(total_time, 60)

def solve():
    try:
        hmInput = input().split()
        if len(hmInput) < 2: return
        h, m = map(int, hmInput)
        
        cookInput = int(input().rstrip())
        cook_time = cookInput
        
        resH, resM = Time(h, m, cook_time)
        
        print(resH, resM)
    except ValueError:
        pass

if __name__ == "__main__":
    solve()

File: 백준/Bronze/2525. 오븐 시계/오븐 시계.cc
#include <stdio.h>

void Time(int h, int m, int* resH, int* resM, int cookTime) {
	int total_time = h * 60 + m + cookTime;

	total_time %= 1440;

	*resH = total_time / 60;
	*resM = total_time % 60;
}

int main(void) {
	int h = 0, m = 0;
	int resH = 0, resM = 0;
	int cookTime = 0;

	if (scanf("%d %d %d", &h, &m, &cookTime) != 3) return -1;

	Time(h, m, &resH, &resM, cookTime);

	printf("%d %d", resH, resM);

	return 0;
}